serenity/master/aarch64/libc.abilist

```diff
@@ -51,7 +51,6 @@
  _exit F
  _longjmp F
  _setjmp F
- _sigsetjmp F
```

Comment:
Neither x86_64 nor riscv64 have this symbol, after it was implemented for aarch64 only `sigsetjmp` remains.